lib/motion-spec/matcher/change.rb

```diff
@@ -0,0 +1,35 @@
+# -*- encoding : utf-8 -*-
+module MotionSpec
+  module Matcher
+    class Change
+      def initialize(change_block)
+        @change_block = change_block
+      end
+
+      def by(amount)
+        @change_amount = amount
+        self
+      end
+
+      def matches?(subject, &expectation_block)
```

Comment:
Unused method argument - `subject`. If it's necessary, use `_` or `_subject` as an argument name to indicate that it won't be used.